src/components/TargetForm/TargetFormContent.js

```diff
@@ -150,6 +153,12 @@ TargetFormContent.propTypes = {
   existent: oneOfType([TARGET_SHAPE, bool]).isRequired,
   actualTopic: string.isRequired,
   setActualTopic: func.isRequired,
+  visible: bool.isRequired,
+  initialValues: objectOf(string),
+};
+
+TargetFormContent.defaultProps = {
+  initialValues: { areaLength: '200 m' },
```

Comment:
WDYT about removing the "m" here so the area is always numeric and you can add the "m" on both cases? Initial values and not?